uber/templates/static_views/js/prereg-common.js

```diff
@@ -0,0 +1,183 @@
+var PREREG_CHECK_INTERVAL = 15 * 60 * 1000;
+var checkCap = function () {
+    $.getJSON('../preregistration/check_prereg', function (check) {
+        if (check.force_refresh) {
+            location.reload();  // Reload the page to prevent registering after reg is closed
+        } else {
+            setTimeout(checkCap, PREREG_CHECK_INTERVAL);
+        }
+    });
+};
+setTimeout(checkCap, PREREG_CHECK_INTERVAL);
+
+{% if c.COLLECT_FULL_ADDRESS %}
```

Comment:
for readability, it might be?  I think move one here might be to at least pull it into its own single file, then once cleaned up, we can decide if we want to break it up further